nucypher/policy/collections.py

```diff
@@ -54,6 +54,7 @@
 
 class TreasureMap:
     ID_LENGTH = 32
+    version = bytes.fromhex("42")  # FIXME: This is a placeholder versioning, while we devise a better data serialization format
```

Comment:
Does that mean we're going the way of having versioned classes instead of a versioned protocol? I guess it provides more flexibility, but there might be problems with processing nested classes, each with its own version.

Reply:
As hinted in the comment, this is just a temporary scheme to facilitate _some_ versioning, reusing the existing serialization approach; it's not intended to be definitive.